docker-compose.yml

```diff
@@ -6,6 +6,21 @@ services:
     restart: unless-stopped
     volumes:
       - pgdata:/var/lib/postgresql/data
+    ports:
```

Comment:
@mbeacom might know if these changes are a good idea or not..

Reply:
Definitely have to sync up with the backend changes that he is making.